test/pi_image_overlay/etc/neon/neon.yaml

```diff
@@ -0,0 +1,181 @@
+play_wav_cmdline: "play %1"
+play_mp3_cmdline: "play %1"
+play_ogg_cmdline: "play %1"
+tts:
+  module: coqui
+  fallback_module: ovos-tts-plugin-mimic
+Audio:
+  backends:
+    OCP:
+      type: ovos_common_play
+      active: true
+      dbus_type: system
+    simple:
+      type: ovos_audio_simple
+      active: true
+    vlc:
+      type: ovos_vlc
+      active: false
+  default-backend: OCP
+audio_parsers:
+  blacklist:
+  - gender
+stt:
+  module: deepspeech_stream_local
+  fallback_module: ovos-stt-plugin-vosk
+  vosk:
+    model: /home/neon/.config/neon/vosk-model-small-en-us-0.15
+  deepspeech_stream_local:
+    model_file: /root/.local/share/neon/deepspeech-0.9.3-models.tflite
+    scorer_file: /root/.local/share/neon/deepspeech-0.9.3-models.scorer
+  ovos-stt-plugin-server:
+    url: https://stt.openvoiceos.com/stt
+confirm_listening: true
+listener:
+  sample_rate: 16000
+  mic_meter_ipc: false
+  record_wake_words: false
+  save_utterances: false
+  wake_word_upload:
+    disable: false
+    url: https://training.mycroft.ai/precise/upload
+  wake_word_enabled: true  # TODO: Deprecate and replace with `continuous_listen`
+  continuous_listen: false
+  VAD:
+    silence_method: vad_and_ratio
+    speech_seconds: 0.1
+    silence_seconds: 0.5
+    before_seconds: 0.5
+    min_seconds: 1
+    max_current_ratio_threshold: 2
+    initial_energy_threshold: 1000.0
+    module: ''
+    ovos-vad-plugin-silero:
+      threshold: 0.2
+    ovos-vad-plugin-webrtcvad:
+      vad_mode: 3
+  mute_during_output: true
+  duck_while_listening: 0.3
+  phoneme_duration: 120
+  multiplier: 1.0
+  energy_ratio: 1.5
+  recording_timeout: 10.0
+  recording_timeout_with_silence: 3.0
+  instant_listen: false
+hotwords:
+  hey mycroft:
+    active: false
+  wake up:
+    active: false
+  hey neon:
+    module: ovos-ww-plugin-vosk
+    listen: true
+    debug: false
+    model_folder: /home/neon/.local/share/neon/vosk-model-small-en-us-0.15
+    samples:
```

Comment:
consider having less transcriptions and playing with the rule, eg use `fuzzy`to see if it improves

```
class MatchRule(str, enum.Enum):
    CONTAINS = "contains"
    EQUALS = "equals"
    STARTS = "starts"
    ENDS = "ends"
    FUZZY = "fuzzy"
    TOKEN_SET_RATIO = "token_set_ratio"
    TOKEN_SORT_RATIO = "token_sort_ratio"
    PARTIAL_TOKEN_SET_RATIO = "partial_token_set_ratio"
    PARTIAL_TOKEN_SORT_RATIO = "partial_token_sort_ratio"
```

Reply:
I'll try `fuzzy`; I just built this list from a couple hours of playing with a mark2 with debug on

Reply:
just like pocketsphinx this will require some tweaking and manual experimenting, but its much easier to handle.

We should even be able to change wake word with a skill by just transcribing a bunch of samples and creating a config with the fuzzy flag

Reply:
This is really an issue for the default config, so I'll fix it in a separate PR

Reply:
https://github.com/NeonGeckoCom/NeonCore/pull/275